frontend/micro-ui/web/micro-ui-internals/packages/modules/dristi/src/configs/UICustomizations.js

```diff
@@ -26,6 +26,13 @@ const partyTypes = {
   "respondent.additional": "Respondent",
 };
 
+const removeInvalidNameParts = (name) => {
+  return name
+    ?.split(" ")
+    .filter((part) => part && !["undefined", "null"].includes(part.toLowerCase()))
+    .join(" ");
+};
```

Comment:
this is extremely odd, additional details shouldn't store these values simply

Reply:
because name came from backend like "test Adv null"